pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -26,12 +26,16 @@ import (
 	"github.com/rook/rook/pkg/operator/ceph/config/keyring"
 	opspec "github.com/rook/rook/pkg/operator/ceph/spec"
 	"github.com/rook/rook/pkg/operator/k8sutil"
+	apps "k8s.io/api/apps/v1"
 	"k8s.io/api/core/v1"
-	extensions "k8s.io/api/extensions/v1beta1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func (c *Cluster) makeDeployment(mgrConfig *mgrConfig) *extensions.Deployment {
+const (
+	mgrDaemonCommand = "ceph-mgr"
```

Comment:
Why has this been added here? I don't see this used in this PR and/or other code parts.

Reply:
it came out of merge, let me double check